martian-filetypes/Cargo.toml

```diff
@@ -19,7 +20,7 @@ flate2 = "1"
 
 [dev-dependencies]
 tempfile = "3"
-criterion = "0.2"
+criterion = "0.3"
```

Comment:
Changes to `dev-dependencies` are not necessary right?

Reply:
Not neccessary, no, but still a good idea if not too inconvenient.  In this case it allows `criterion` to use the same `rand` as everything else, which should make builds a little faster.